pages/apim/3.x/installation-guide/upgrades/3.18.8/README.adoc

```diff
@@ -0,0 +1,10 @@
+= Upgrade to 3.18.8
+
+== Breaking changes
+
+=== Return type of /apis/{apiId}/subscribers has changed
+
+In order to improve performances while fetching the subscribers of an API, internal methods have changed and the output has evolved.
+When calling `/organizations/{orgId}/environments/{envId}/apis/{apiId}/subscribers`, you will now get a list of `ApplicationListItem` instead of `ApplicationEntity`.
```

Comment:
```suggestion
As a result, a call to `/organizations/{orgId}/environments/{envId}/apis/{apiId}/subscribers` now returns a list of `ApplicationListItem` instead of `ApplicationEntity`.
```